project/npda/views/home.py

```diff
@@ -165,47 +165,13 @@ def view_preference(request):
         new_session_fields = get_new_session_fields(
             request.user, pz_code
         )  # includes a validation step
-
+    
     request.session.update(new_session_fields)
-    context = {
-        "view_preference": view_preference,
-        "chosen_pdu": pz_code,
-        "pdu_choices": request.session["pdu_choices"],
-    }
-
-    response = render(
-        request, template_name="partials/view_preference.html", context=context
-    )
-
-    patients_list_view_url = reverse("patients")
-    submissions_list_view_url = reverse("submissions")
-    npdauser_list_view_url = reverse("npda_users")
-    dashboard_url = reverse("dashboard")
-
-    trigger_client_event(
```

Comment:
I was seeing bugs because my new UI in the patient table view wasn't re-rendering when I changed the audit year or PDU switcher. That's because it wasn't part of the patient table reloaded by HTMX.

Honestly I think a full page reload is the safer thing to do here anyway, we won't have to remember to keep adding to this list